services/cd-service/pkg/service/git_test.go

```diff
@@ -442,43 +442,6 @@ func TestGetProductDBFailureCases(t *testing.T) {
 			timestamp:     &ts,
 			expectedErr:   errMatcher{"Can not have both an environment and environmentGroup to get the product summary for"},
 		},
-		{
-			Name:      "invalid environment used",
```

Comment:
If an invalid environment is sent to GetProductSummary, we can just return an empty response. No need for returning error.